include/deal.II/fe/fe_update_flags.h

```diff
@@ -240,114 +302,143 @@ inline StreamType &
 operator<<(StreamType &s, const UpdateFlags u)
 {
   s << " UpdateFlags|";
-  if (u & update_values)
+  if (u & UpdateFlags::update_values)
```

Comment:
Shouldn't this use `contains` as well? Out of curiosity, why do you need the classname qualification?

Reply:
Yes, fixed. I used the class name due to some problems with the order of declarations/definitions. Using the class static variables worked better.